vector/src/main/java/fr/gouv/tchap/features/login/TchapLoginViewModel.kt

```diff
@@ -250,9 +237,9 @@ class TchapLoginViewModel @AssistedInject constructor(
         reAuthHelper.data = action.password
         currentJob = executeRegistrationStep {
             it.createAccount(
```

Comment:
We may add a comment here to explain:
- Tchap is not expected username
- this first register request will linked the pwd with the returned session id (used in the following steps)